diktat-rules/src/test/resources/test/paragraph3/file_structure/OtherCommentsExpected.kt

```diff
@@ -0,0 +1,11 @@
+/**
+ * This is an example
+ */
+
+// some notes on this file
```

Comment:
Maybe it's better to add comments into KDoc?

Reply:
I check that EOL comments are moved correctly here, why would I need KDoc instead/

Reply:
I thought it would be better to have one KDOC than KDOC and EOL, but I think so well)